src/jbi/services.py

```diff
@@ -51,9 +51,20 @@ def wrapped_func(*args, **kwargs):
         return wrapped_func
 
 
+class CustomJira(Jira):
+    def get_permissions(self, permissions, project_key, **kwargs):
+        # Waiting for https://github.com/atlassian-api/atlassian-python-api/pull/1016/
+        # to be released.
+        url = self.resource_url("mypermissions")
+        params = {"permissions": permissions}
+        if project_key:
+            params["projectKey"] = project_key
+        return self.get(url, params=params)
+
+
```

Comment:
I think we can revert this commit and use the atlassian package

Reply:
What does `poetry show | grep atlassian` show?

`atlassian-python-api 3.25.0    Python Atlassian REST API Wrapper`

If you have a fork or something to be used, `poetry.lock`  would have to point to it.
Otherwise, try `make clean` and `make start` ?

Reply:
> jira-bugzilla-integration bsieber$ poetry show | grep atlassian
atlassian-python-api 3.25.0    Python Atlassian REST API Wrapper

Reply:
```
> jira-bugzilla-integration bsieber$ make start
POETRY_VIRTUALENVS_IN_PROJECT=1 poetry install --no-root
Installing dependencies from lock file

No dependencies to install or update
touch .venv/.install.stamp
touch: .venv/.install.stamp: No such file or directory
make: *** [.venv/.install.stamp] Error 1
```

Reply:
To be discussed outside of this thread